llama-index-core/llama_index/core/vector_stores/simple.py

```diff
@@ -91,12 +91,19 @@ def _process_filter_match(
         filter_matches_list = []
         for filter_ in filter_list:
             filter_matches = True
-
-            filter_matches = _process_filter_match(
-                operator=filter_.operator,
-                value=filter_.value,
-                metadata_value=metadata.get(filter_.key, None),
-            )
+            metadata_value = metadata.get(filter_.key, None)
+            if filter_.operator == FilterOperator.IS_EMPTY:
```

Comment:
I think `_process_filter_match` is for the matching with non-null value, but the is_empty don't use the value. So i just put it here. If it does makes sense then i can just refactor it.